py-rse/packaging.Rmd

```diff
@@ -889,36 +834,20 @@ Contributors
 - Sami Virtanen <sami@zipf.org>
 ```
 
-FIXME: Does the installation section of the README need to list the dependencies
-(i.e. `numpy`, `scipy`, `pytest` etc)? 
+### Creating a Web Page for Documentation {#packaging-sphinx}
 
-For simple packages,
-carefully written docstrings and READMEs
-are sufficient to describe the package
-and is infinitely better than not documenting your package at all.
-As you code base grows larger,
-it will become convenient to complement these manually written sections
-with automatically generated content,
+Docstrings and READMEs are sufficient to describe most simple packages,
+and are infinitely better than no documentation at all.
+As our code base grows larger,
+though,
+we will want to complement these manually written sections with automatically generated content,
 references between functions,
 and search functionality.
-We will see how to do this next.
-
-### Creating a web page for documentation {#package-py-sphinx}
 
-If you've ever viewed the online documentation for a large Python package,
-it's likely to have been prepared using [Sphinx](http://www.sphinx-doc.org/en/master/),
-a package for writing software documentation and generating web packages.
-When combined with [Read The Docs](https://readthedocs.org) --
-a free service for hosting online documentation
-that understands documents written to support Sphinx --
-it is an extremely useful way to build documentation and an online home for your package.
-Both tools recognize reStructuredText files.
-
-We'll start by putting together a minimal set of documentation
-that includes the `README.rst` file we just created
-and the docstrings we've written along the way.
-Sphinx makes it relatively straightforwrd to get started,
-using the `quickstart` tool.
+The online documentation for most large Python packages
+is generated using a tool called [Sphinx](http://www.sphinx-doc.org/en/master/),
+which is often used in combination with [Read The Docs](https://readthedocs.org),
```

Comment:
Should the links to Sphinx and Read the Docs be in `links.md` rather than embedded in the text?